<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ardusub Frame Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/prefix1/icons/icon.png</normaloff>:/new/prefix1/icons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 20px; border-radius:15px;
background-color: rgb(39, 43, 44);
 </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>620</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spin_motors">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>47</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_frame_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 10pt;
font-weight: bold;
color: rgb(148, 148, 148);</string>
     </property>
     <property name="text">
      <string>ROV Frame name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtFrameName">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_number_motors">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 10pt;
font-weight: bold;
color: rgb(148, 148, 148);</string>
     </property>
     <property name="text">
      <string>Number of Motors</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 30px; border-radius:15px;
background-color: rgb(89, 96, 102);
font-size: 12pt;
font-weight: bold;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QListWidget" name="motors_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>620</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 245, 245);

</string>
    </property>
    <property name="movement">
     <enum>QListView::Free</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 30px; border-radius:15px;
background-color: rgb(89, 96, 102);
font-size: 12pt;
font-weight: bold;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>frame</zorder>
   <zorder>btnGenerate</zorder>
   <zorder>motors_list</zorder>
   <zorder>btnAbout</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
